.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
Monthly would be a little too often IMO. Can we set this to quarterly?

Reply:
Also, can we control *when* this happens? It might be good to set it so that it doesn't come too close to our release dates.

Reply:
No, unfortunately, that's not possible to configure. `"monthly"` is the longest interval currently available, and Dependabot chooses the first day of each month ([docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval)).

There are two things that will reduce the likelihood of receiving a disruptive number of PRs on the first of each month:

1. Dependabot submits an aggregate PR for each action, not a PR for each individual line that will change. For example, there would be a single PR to update `actions/checkout` to the latest version across all occurrences in all YAML files.
2. This repo exclusively uses major version identifiers (like `actions/checkout@v4`, as opposed to `@4.1.1`) and relies on ~5 actions by my count; Dependabot would therefore only submit a PR when version 5 is released.